apps/todolist-mongo-go/Dockerfile

```diff
@@ -1,18 +1,18 @@
 #FROM registry.access.redhat.com/ubi8/ubi-minimal
-FROM golang:1.17.8-alpine AS build-env
+FROM golang:1.23-alpine AS build-env
 RUN mkdir /build
 WORKDIR /build
-COPY apps/todolist-mongo-go/*.go .
-COPY apps/todolist-mongo-go/go.mod .
-COPY apps/todolist-mongo-go/go.sum .
+COPY *.go .
+COPY go.mod .
+COPY go.sum .
```

Comment:
Any reason for this change? This is a breaking change if anything else depended on prior behavior.

Reply:
the src is no longer under apps when building this container.